scripts/output/comparison/compareScenarios2.R

```diff
@@ -0,0 +1,140 @@
+# |  REMIND License Exception, version 1.0 (see LICENSE file).
+# |  Contact: remind@pik-potsdam.de
+# ---- Define set of runs that will be compared ----
+
+if (exists("outputdirs")) {
+  # This is the case if this script was called via Rscript output.R
+  listofruns <- list(list(
+    period = "both",
+    set = format(Sys.time(), "%Y-%m-%d_%H.%M.%S"),
+    dirs = outputdirs))
+} else {
+  # This is the case if this script was called directly via Rscript
+  listofruns <- list(
+    list(
+      period = "both",
+      set = "cpl-Base",
+      dirs = c("C_SDP-Base-rem-5","C_SSP1-Base-rem-5","C_SSP2-Base-rem-5","C_SSP5-Base-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg900",
+      dirs = c("C_SDP-PkBudg900-rem-5","C_SSP1-PkBudg900-rem-5","C_SSP2-PkBudg900-rem-5","C_SSP5-PkBudg900-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg1100",
+      dirs = c("C_SDP-PkBudg1100-rem-5","C_SSP1-PkBudg1100-rem-5","C_SSP2-PkBudg1100-rem-5","C_SSP5-PkBudg1100-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg1300",
+      dirs = c("C_SDP-PkBudg1300-rem-5","C_SSP1-PkBudg1300-rem-5","C_SSP2-PkBudg1300-rem-5","C_SSP5-PkBudg1300-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-NPi",
+      dirs = c("C_SDP-NPi-rem-5","C_SSP1-NPi-rem-5","C_SSP2-NPi-rem-5","C_SSP5-NPi-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SDP",
+      dirs = c("C_SDP-Base-rem-5","C_SDP-NPi-rem-5","C_SDP-PkBudg1300-rem-5","C_SDP-PkBudg1100-rem-5","C_SDP-PkBudg1000-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP1",
+      dirs = c("C_SSP1-Base-rem-5","C_SSP1-NPi-rem-5","C_SSP1-PkBudg1300-rem-5","C_SSP1-PkBudg1100-rem-5","C_SSP1-PkBudg900-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP2",
+      dirs = c("C_SSP2-Base-rem-5","C_SSP2-NPi-rem-5","C_SSP2-PkBudg1300-rem-5","C_SSP2-PkBudg1100-rem-5","C_SSP2-PkBudg900-rem-5","C_SSP2-NDC-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP5",
+      dirs = c("C_SSP5-Base-rem-5","C_SSP5-NPi-rem-5","C_SSP5-PkBudg1300-rem-5","C_SSP5-PkBudg1100-rem-5","C_SSP5-PkBudg900-rem-5")))
+}
+
+# remove the NULL element
+listofruns <- listofruns[!sapply(listofruns, is.null)]
+
+# if no path in "dirs" starts with "output/" insert it at the beginning
+# this is the case if listofruns was created in the lower case above !exists("outputdirs"), i.e. if this script was not called via Rscript output.R
+for (i in 1:length(listofruns)) {
+  if (!any(grepl("output/", listofruns[[i]]$dirs))) {
+    listofruns[[i]]$dirs <- paste0("output/", listofruns[[i]]$dirs)
+  }
+}
+
+# ---- Start compareScenarios either on the cluster or locally ----
+
+start_comp <- function(outputdirs,
+                       shortTerm,
+                       outfilename,
+                       regionList,
+                       mainReg) {
+  if (!exists("slurmConfig")) {
+    slurmConfig <- "--qos=standby"
+  }
+  jobname <- paste0(
+      "cs2",
```

Comment:
Is there a good reason not to call it `compScen` as well? I mean, as you said, for the user, nothing needs to change… If you stick to `cs2`, please add it in the [.gitignore](https://github.com/remindmodel/remind/blob/develop/.gitignore#L27) in the same way.

Reply:
Well, it's not necessarily a "good" reason, but I usually use shorthand cs2 for compareSceanrios2 and it might be better to have a different name for the document to indicate that it is not coming from the old compareScenarios. But, yes, if `cs2` is used it has to go into gitignore

Reply:
I am certainly not the one who should decide that :)